backend/app/controllers/api/PagesController.scala

```diff
@@ -100,9 +100,9 @@ class PagesController(val controllerComponents: AuthControllerComponents, manife
       hlText.highlights.map(span => (lang, span))
     }.groupBy(_._2).toList.map { case (lang, commonSpans) =>
       commonSpans.head
-    }.groupBy(_._1).mapValues(_.map(_._2)).filter { case (k, v) =>
+    }.groupBy(_._1).view.mapValues(_.map(_._2)).filter { case (k, v) =>
```

Comment:
So `mapValues` is deprecated, so I added `.view` as suggested by the warning. We use enough of this call that I think the tests would catch if it's semantically different somehow but maybe worth additional testing?